capstone/src/main/java/com/google/univiz/api/MapData.java

```diff
@@ -1 +1,13 @@
 package com.google.univiz.api;
+
+/*
+ * The MapData interface is responsible for all data management/processing specific for the maps
+ * feature of the UniViz webapp.
+ */
+public interface MapData {
+  /*
+   * Takes a list of college names and returns a CollegeMapsData object to the caller. This object
+   * contains all data relevant to the Maps feature.
+   */
+  CollegeMapsData getMapData(List<String> collegeNames);
```

Comment:
I expected to see the `CollegeMapsData` class definition in this PR too. If you're not sure about its fields, start by adding an empty class and fill it out in a followup PR.

Reply:
will do!